process/hyper_block_processor.go

```diff
@@ -51,8 +90,60 @@ func (hbp *hyperBlockProcessor) Process(hyperBlock *hyperBlock.HyperBlock) (*sch
 		return nil, err
 	}
 
-	avroHyperBlock.ShardBlocks = shardBlocks
-	avroHyperBlock.Transactions = txs
-	avroHyperBlock.EpochStartInfo = epochStartInfo
-	return avroHyperBlock, nil
+	return &schemaV2.HyperBlock{
+		Hash:                   hash,
+		PrevBlockHash:          prevBlockHash,
+		StateRootHash:          stateRootHash,
+		Nonce:                  int64(hyperBlock.Nonce),
+		Round:                  int64(hyperBlock.Round),
+		Epoch:                  int32(hyperBlock.Epoch),
+		NumTxs:                 int32(hyperBlock.NumTxs),
+		AccumulatedFees:        accumulatedFees,
+		DeveloperFees:          developerFees,
+		AccumulatedFeesInEpoch: accumulatedFeesInEpoch,
+		DeveloperFeesInEpoch:   developerFeesInEpoch,
+		Timestamp:              int64(hyperBlock.Timestamp),
+		EpochStartInfo:         epochStartInfoOrNil(epochStartInfo),
+		ShardBlocks:            shardBlocksOrNil(shardBlocks),
+		Transactions:           txsOrNil(txs),
+		StateChanges:           nil,
```

Comment:
Yes, it is left on purpose, since there is no official support on either devnet/mainet for altered accounts.
This field is defined in avro schema as possible nil, so it's fine